rcqt.so/src/debPackageInfo.cpp

```diff
@@ -1,24 +1,68 @@
 #include <iostream>
+#include <vector>
 #include <sstream>
 #include <fstream>
 #include "include/debPackageInfo.h"
 
+std::vector<std::string> dependency_patterns{"(=", "(>="};
+
+std::string findPattern(std::string word){
```

Comment:
Should we move this function to utils ? Or is this really specific to deb ?

Reply:
did not see any such versions on rhel or sles. so currently kept in ubuntu/deb